IOMC/RandomEngine/python/IOMC_cff.py

```diff
@@ -172,6 +172,11 @@
 
 randomEngineStateProducer = cms.EDProducer("RandomEngineStateProducer")
 
+from Configuration.Eras.Modifier_run2_GEM_2017_MCTest_cff import run2_GEM_2017_MCTest
```

Comment:
generically for anything related to MC only - could we not already use the run2_2017 era?